src/components/Message/MessageSimple/utils/renderText.tsx

```diff
@@ -212,13 +210,15 @@ export const renderText = <
       propOnPress?.({
         emitter: 'textMention',
         event,
+        mentionedUserName: node.content[0]?.content,
```

Comment:
end user will need to loop through `message.mentioned_users` to identify exact user id